### PR TITLE
Add main product page

### PR DESCRIPTION
**Pull request by DavidBiddle**
Includes:
- Design system stuff from NPM and a couple of build tasks
- HTML structure for layout and main page
- Styles from DI repo
- Placeholder pages for the accessibility, privacy and cookie policies
- Fix for a small issue where our text breaks the signon layout
Doesn't include:
- Content for the accessibility, privacy and cookie policies
- Instructions for running the build (we don't have a README yet, separate PR incoming)

### CODE REVIEW

**Review by thomasiles — APPROVED** [review]
Awesome!

File: source/index.html.erb [hunk 1]
```diff
@@ -1,23 +1,196 @@
...
+            focusable="false"
+          >
+            <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"></path>
+          </svg>
+        </a>
+        <span class="hero-alternative-action">
+          or &nbsp;<a class="govuk-link" href="#"
+            >sign in if you're a private beta partner</a
+          >
+        </span>
```
Comment (danielburnley) [inline source/index.html.erb:45]: We can probably remove this for now

**Review by danielburnley — APPROVED** [review]
Works locally - marked a part we can probably remove but we can do that in a follow up PR

File: source/index.html.erb (hunk 1 above)
Comment (DavidBiddle) [reply source/index.html.erb]: Done 🎉

**Pull request merged**